Components/OperationManagerPatches.cs

```diff
@@ -10,8 +10,8 @@ public class OperationManagerPatches
     {
         public static void Patch()
         {
-            Harmony.MakeRET(typeof(OperationManager), "isAimeOffline");
-            Harmony.MakeRET(typeof(OperationManager), "isAimeLoginDisable");
+            Harmony.MakeRET(typeof(OperationManager), "isAimeOffline", false);
```

Comment:
retVal is false by default, why add these explicitly?

Reply:
That makes a lot more sense now. Reverted.